cli/azd/pkg/tools/azcli/azcli.go

```diff
@@ -55,7 +55,7 @@ type AzCli interface {
 	// the interactive browser login flow happens. In the case of a device code login, the message is written to the
 	// `deviceCodeWriter`.
 	Login(ctx context.Context, useDeviceCode bool, deviceCodeWriter io.Writer) error
-	LoginAcr(ctx context.Context, subscriptionId string, loginServer string) error
+	LoginAcr(ctx context.Context, subscriptionId string, resourceGroup string, loginServer string) error
```

Comment:
Same question here... Would it be possible to avoid `resourceGroup`? This would definitely make it harder to use given that we now need `{loginServer, resourceGroup}` as the tuple for resolving the resource

Reply:
This is the same issue we saw the the keyvault.  The `az` commands perform list by subscription and find the registry by name, then perform the `GetCredentials` command using the found registry/resource group from the list.

We could offer to do the same but in our use case we already have the resource group.